src/test/scala/io/github/spark_redshift_community/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -442,6 +442,28 @@ class RedshiftSourceSuite
     mockRedshift.verifyThatExpectedQueriesWereIssued(expectedCommands)
   }
 
+  test("Include Column List adds the schema columns to the COPY query") {
```

Comment:
Could you please write a second small test near this one showing the difference when include_column_list = false ?